### ISSUE TITLE
Turn off Log-in Prompt in qml after configuring for Portal.

### ISSUE THREAD

**Issue by TuckerLonginotti** [Esri/Survey123Community#589]
I have configured my Survey123 app to go directly to my portal when you click Sign In when opening Survey123.  I did this by replacing 'url' and 'name' in PortalsList.qml under ListModel and function read().  Also, updating kDefaultPortalUrl and function readSettings() in Portal.qml with my portal information.

My issue is that when you click Sign In, it launches the keyboard and sign in dialog box with the exact portal url.  This is an issue because you have to cancel out and click "Sign in using [your portal...]" for the dialog to appear.  Clicking the button populates the correct url that incorporates the authorization needed.  

Basically, I need to find the code that prompts the "Authentication Required" dialog box after you hit Sign In and turn that off.  I need the user to have to click "Sign in to [Portal]" on the Portal Sign In page in order to get the authentication required.  Below is a image of the dialog I would not like to see after clicking Sign In and being directed to the Portal Sign in page.

![capture](https://cloud.githubusercontent.com/assets/15017826/21824436/a5dac9fa-d73c-11e6-8ae1-0e9128064582.JPG)
